VST/tests/vst_instrument_test.ck

```diff
@@ -0,0 +1,28 @@
+"C:/VSTPlugIns/Serum_x64.dll" => string vst_path;  // Must use absolute path.
```

Comment:
Are there MIT-licensed VST instrument(s) we could use here instead of proprietary ones?

Reply:
What do you think of Tal-NoiseMaker? It's [GPL](https://tal-software.com/downloads/docs/TAL%20Noisemaker%20User%20Guide%201.0.pdf) and working with my test. https://tal-software.com/products/tal-noisemaker

Reply:
I've add the DLL in this commit https://github.com/ccrma/chugins/pull/47/commits/79c3e59901c3cfc99e0c35775bc478fcd343d5f3